### PR TITLE
Release 162.0.0

### PR DESCRIPTION
**Pull request by jiexi**
Releases `queued-request-controller` methods array constructor param to callback constructor param update

### CODE REVIEW

**Comment by jiexi** [comment]
@metamaskbot publish-preview

File: packages/queued-request-controller/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@metamask/queued-request-controller",
-  "version": "0.12.0",
+  "version": "1.0.0",
```
Comment (adonesky1) [inline packages/queued-request-controller/package.json:3]: oooo he did it. Stepping into the major leagues 🤣

File: packages/queued-request-controller/CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [1.0.0]
+
+### Changed
+
+- **BREAKING:** `QueuedRequestController` constructor no longer accepts the `methodsRequiringNetworkSwitch` array param. It's now replaced with the `shouldRequestSwitchNetwork` function param which should return true when a request requires the globally selected network to match that of the dapp. ([#4423](https://github.com/MetaMask/core/pull/4423))
```
Comment (adonesky1) [inline packages/queued-request-controller/CHANGELOG.md:14]: total nit
```suggestion
- **BREAKING:** `QueuedRequestController` constructor no longer accepts the `methodsRequiringNetworkSwitch` array param. It's now replaced with the `shouldRequestSwitchNetwork` function param which should return true when a request requires the globally selected network to match that of the dapp from which the request originated. ([#4423](https://github.com/MetaMask/core/pull/4423))
```

**Review by adonesky1 — APPROVED** [review]
LGTM. Just a couple nits

File: packages/queued-request-controller/package.json (hunk 1 above)
Comment (jiexi) [reply packages/queued-request-controller/package.json:3]: haha can change this back to minor if preferred

Comment (adonesky1) [reply packages/queued-request-controller/package.json:3]: Nah its time for this boy to become a man

**Review by adonesky1 — APPROVED** [review]

**Pull request merged**